app/views/vocabularies/new.html.erb

```diff
@@ -3,4 +3,4 @@
 <% end %>
 <h1> Create New Vocabulary </h1>
 
-<%= render :partial => "form", :locals => { :vocabulary => @vocabulary } %>
+<%= render :partial => "form", :locals => { :relationship => @relationship } %>
```

Comment:
Yeah, I didn't even notice I changed this. I must have thought I was in the relationships new page. This is what broke a ton of tests locally. 👍